src/Components/Modals/EditAdditionalRepositoriesModal.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import propTypes from 'prop-types';
+import { Modal, ModalVariant, Button } from '@patternfly/react-core';
+import EditAdditionalRepositoriesTable from '../EditAdditionalRepositoriesTable/EditAdditionalRepositoriesTable';
+
+const EditAdditionalRepositoriesModal = (props) => {
+  const { handleModalToggle, isOpen, repositories } = props;
+
+  const editAdditionalRepositoriesDescription =
+    'The core repositories for your operating system version, for example BaseOS and AppStream, are always enabled and do not need to be explicitly added to the activation key.';
+
+  /* Modal Return with attributes */
+
+  return (
+    <React.Fragment>
+      <Modal
+        variant={ModalVariant.large}
+        title="Edit additional repositories"
+        description={editAdditionalRepositoriesDescription}
+        isOpen={isOpen}
+        onClose={handleModalToggle}
+        actions={[
+          <Button
+            key="Save changes"
+            variant="primary"
+            onClick={handleModalToggle}
+            isDisabled
```

Comment:
is this save changes button supposed to permanently be disabled?

Reply:
I had it disabled as it was in the wireframe sketch , but i took that attribute out for future edits !

Reply:
Just to clarify, for this story we aren't really doing anything with this modal except showing the data and sorting/searching.  The functionality behind that Save button will come in a later story.

At this point it doesn't matter if Save is enabled or disabled, so I'll let you all decide how you want it.  I just wanted to provide the context.